frontend_v2/src/app/components/challenge/challengemanage/challengemanage.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+
+// import service
+import { EndpointsService } from '../../../services/endpoints.service';
+import { AuthService } from '../../../services/auth.service';
+import { ChallengeService } from '../../../services/challenge.service';
+import { ApiService } from '../../../services/api.service';
+import { GlobalService } from '../../../services/global.service';
+
+@Component({
+  selector: 'app-challengemanage',
+  templateUrl: './challengemanage.component.html',
+  styleUrls: ['./challengemanage.component.scss']
+})
+export class ChallengemanageComponent implements OnInit, OnDestroy {
+
+  /**
+   * store worker logs
+   */
+  workerLogs = [];
+
+  /**
+   * Challenge object
+   */
+  challenge: any;
+
+  /**
+   * Is user logged in
+   */
+  isLoggedIn = false;
+
+  /**
+   * An interval for fetching the leaderboard data in every 5 seconds
+   */
+  pollingInterval: any;
+
+  constructor(private endpointService: EndpointsService, private authService: AuthService,
+    private challengeService: ChallengeService, private apiService: ApiService,
+    private globalService: GlobalService) { }
+
+  ngOnInit() {
+    if (this.authService.isLoggedIn()) {
+      this.isLoggedIn = true;
+    }
+    this.challengeService.currentChallenge.subscribe(challenge => {
+      this.challenge = challenge;
+    });
+    this.fetchWorkerLogs();
+    this.startLoadingLogs();
+  }
+
+  // API call to manage the worker from UI.
+  // Response data will be like: {action: "Success" or "Failure", error: <String to include only if action is Failure.>}
+  manageWorker(action) {
+    const SELF = this;
+    const API_PATH = SELF.endpointService.manageWorkerURL(SELF.challenge['id'], action);
+    const BODY = JSON.stringify('');
+    SELF.apiService.putUrl(API_PATH, BODY).subscribe(
+      data => {
+        if (data[action] === 'Success') {
+          SELF.globalService.showToast('success', 'Worker(s) ' + action + 'ed succesfully.', 5);
```

Comment:
In this toast, padding was off from what I saw in the demo GIF.

Reply:
Hey @lunayach thanks for pointing this out. When i pulled the toast changes, i found that it's not working as expected so now i have modified it a little bit. Attaching the new GIF here.